sites/_variables.scss

```diff
@@ -62,6 +64,12 @@ Left menu variables
 $menu-background: #fff;
 $menu-colour: #555;
 
+// Spaces variables
+$space-0: 0px;
+$space-sm: 5px;
+$space-md: 15px;
+$space-lg: 30px;
```

Comment:
Don't we already have those variable defined in wet-boew or bootstrap? Or an equivalent for padding and margin? if so please use those variable instead of creating new one